app/src/main/java/org/simple/clinic/summary/compose/StatinNudgeView.kt

```diff
@@ -28,58 +28,61 @@ import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import org.simple.clinic.R
 import org.simple.clinic.common.ui.theme.SimpleTheme
+import org.simple.clinic.cvdrisk.StatinInfo
 import org.simple.clinic.util.toAnnotatedString
 
 
 @Composable
 fun StatinNudge(
-    isVisible: Boolean,
+    statinInfo: StatinInfo?,
     modifier: Modifier = Modifier
 ) {
-  AnimatedVisibility(
-      visible = isVisible,
-      enter = expandVertically(
-          animationSpec = tween(500),
-          expandFrom = Alignment.Top
-      ),
-      exit = shrinkVertically(animationSpec = tween(500))
-  ) {
-    Card(
-        modifier = modifier
+  if (statinInfo != null) {
```

Comment:
Any reason for using the `if` condition here? Was there a issue with the UI without this?

Reply:
Remove null check